job/scriptsGen/generateScript.js

```diff
@@ -246,6 +252,58 @@ function _createBootScript(bag, next) {
   );
 }
 
+function _generateBuildScriptFromTemplate(bag, next) {
```

Comment:
Note: If there are multiple container TASKs, this file will get generated multiple times unnecessarily. This shouldn't cause any significant loss in performance, so I'm keep this as-is for the sake of simplicity.